### PR TITLE
Update dist directory

### PR DESCRIPTION
**Pull request by singingwolfboy**
To make this pull request, I did the following things:

* Cloned the repository
* `yarn install`
* Discovered that [the project depends on `html-webpack-plugin`](https://github.com/pagarme/react-event-components/blob/master/webpack.config.js#L19), added it to `package.json`
* `yarn build`
* Commit all changes
* Push up to GitHub and make pull request

Notably, this updates the files in the `dist` directory to no longer contain references to `React.PropTypes`, which is the problem I was running into in #46.

### CODE REVIEW

**Comment by felquis** [comment]
Just started reviewing :octocat:

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
   "main": "dist/react-event-components.js",
   "devDependencies": {
     "gh-pages": "1.0.0",
+    "html-webpack-plugin": "^2.30.1",
```
Comment (felquis) [inline package.json]: Great discovery, I remove all carets from our package.json

**Comment by felquis** [comment]
I noticed you are much faster than us

**Comment by felquis** [comment]
I created another PR just to split your commit, because "update dist" doesn't mention adding the missing dep!

@derekstavis will help me publish it to npm today, we are at UTC-3 here in Brazil.

**Comment by derekstavis** [comment]
superseded by #48

**Pull request closed without merging**